### DISCUSSION TITLE
Missing "Explore" in nav bar

### DISCUSSION THREAD

**Discussion by 403-html** [community/community#60117] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

out of habit I like to check what is trending and explore the designs/apps/scripts/ideas, there used to be a cool button on the navbar, now it is not there and it took me a while to find it, can it be restored?

**Comment by 3urobeat** [comment]
Just click on the hamburger menu in the top left and then on "Explore" at the bottom. It's just 2 clicks :D

**Reply by 403-html** [reply]
back then it was 1, UX downgrade :D

**Reply by 3urobeat** [reply]
Welllll yeah but no  
I don't think putting a menu which is not as often used as others behind one more click isn't really that much of a downgrade

**Reply by 403-html** [reply]
How often do you open "Issues" or "Pull requests"? Tbh I go there when there's notification that something is cooking up down there. On the other hand "Explore" is like daily routine as trending stuff can change within 24h.

**Reply by 3urobeat** [reply]
Pretty much regularly but you can't compare a repository specific menu with a global menu  
However I'm no UX designer, so :shrug:

**Reply by 403-html** [reply]
> you can't compare a repository specific menu with a global menu

![image](https://github.com/community/community/assets/57900160/36407a67-44ae-45a8-b45a-a1086b6cd598)

But they are all in the global navigation bar (in the upper right corner), so... it's not repository specific, and Explore was there too (center, but it can be easily made as icon button as rest too), so it lacks the functionality that was introduced there, and frankly I think the Explore page will lose users because it's no longer eye-catching, but it gave a lot of value for learning stuff and discovering new tech

**Reply by 3urobeat** [reply]
Oh interesting, I didn't even see these two new PR and Issue menus. Sorry  
I guess you are right then, an Explore button at the same place would make sense
